vpd-tool/src/vpd_tool.cpp

```diff
@@ -74,4 +74,32 @@ int VpdTool::readKeyword(const std::string& i_vpdPath,
     }
     return l_rc;
 }
+
+int VpdTool::dumpObject(const std::string& i_fruPath) const noexcept
+{
+    int l_rc{constants::FAILURE};
+    try
+    {
+        const nlohmann::json l_resultJson = getFruProperties(i_fruPath);
+
+        utils::printJson(l_resultJson);
+        l_rc = constants::SUCCESS;
+    }
+    catch (std::exception& l_ex)
+    {
+        std::cerr << "Dump Object failed for FRU: " << i_fruPath
```

Comment:
add todo for enable log when verbose is enabled

Reply:
Done